scripts/run-test-suite

```diff
@@ -27,8 +26,10 @@ riot_hashes=$(riot list --hash-only $RIOT_PATTERN | sort | circleci tests split)
 
 all_passed=false
 for hash in $riot_hashes; do
-    $DDTEST_CMD riot -v run --exitfirst --pass-env -s $hash $DDTRACE_FLAG $COVERAGE_FLAG
-    [[ $? > 0 ]] && circleci step halt && exit $?
+    if $DDTEST_CMD riot -v run --exitfirst --pass-env -s $hash $DDTRACE_FLAG $COVERAGE_FLAG; then
+        circleci step halt
+        exit 1
+    fi
```

Comment:
I'm not quite sure how this would work with the `circleci` command, but I'd start optimistically with
```
all_passed=true
```
and
```suggestion
    if $DDTEST_CMD riot -v run --exitfirst --pass-env -s $hash $DDTRACE_FLAG $COVERAGE_FLAG
    then
        circleci step halt
        all_passed=false
    fi
```
assuming we'd want to run all hashes